pkgs/applications/networking/cluster/nomad/generic.nix

```diff
@@ -13,22 +19,25 @@ buildGoPackage rec {
     repo = pname;
     inherit rev sha256;
   };
+  inherit patches;
 
   # ui:
   #  Nomad release commits include the compiled version of the UI, but the file
   #  is only included if we build with the ui tag.
   # nonvidia:
   #  We disable Nvidia GPU scheduling on Linux, as it doesn't work there:
   #  Ref: https://github.com/hashicorp/nomad/issues/5535
-  preBuild = let
-    tags = ["ui"]
-      ++ stdenv.lib.optional stdenv.isLinux "nonvidia";
-    tagsString = stdenv.lib.concatStringsSep " " tags;
-  in ''
-    export buildFlagsArray=(
-      -tags="${tagsString}"
-    )
- '';
+  preBuild =
```

Comment:
I hate to be _that_ person but could you undo the formatting changes?

Reply:
Yup.